modules/database/database_addons.bal

```diff
@@ -15,34 +12,54 @@ import ballerina/io;
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import ballerina/sql;
+import ballerina/io;
+
 public type IdSQL record {|
     int id;
 |};
 
+# Get experiment info
+#
+# + experimentId - experiment id
+# + return - experiment info
+public isolated transactional function getExperimentInfo(int experimentId) returns Experiment|error {
```

Comment:
This seems like duplicated code. There should already be a function to fetch an experiment from the database in the database module.